fixtures/cloudfour.com-patterns/.storybook/main.js

```diff
@@ -0,0 +1,59 @@
+const { resolve } = require('path');
```

Comment:
Per @spaceninja's previous comment/question, this is a Cloud Four project, but the file's being used as a fixture, and therefore I figure not under review for this PR. Correct?

Reply:
![](https://user-images.githubusercontent.com/13206945/77784522-e498cf80-7017-11ea-9d44-a76354181ec3.png)

The files in these folders can be ignored